### PR TITLE
feat: release psp-v2022 202304.0

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

File: bundles/psp-v2022/psp-capabilities.yaml [hunk 1]
```diff
@@ -55,6 +57,18 @@ spec:
...
+    - FSETID
+    - KILL
+    - MKNOD
+    - NET_BIND_SERVICE
+    - SETFCAP
+    - SETGID
+    - SETPCAP
+    - SETUID
+    - SYS_CHROOT
+    # requiredDropCapabilities:
```
Comment (g-awmalik) [inline bundles/psp-v2022/psp-capabilities.yaml:73]: Do we need this?

File: bundles/psp-v2022/psp-flexvolume-drivers.yaml [hunk 2]
```diff
@@ -39,7 +39,9 @@ spec:
...
         kinds:
           - Pod
-  parameters:
-    allowedFlexVolumes: #[]
-      - driver: example/lvm
-      - driver: example/cifs
+  # K8sPSPFlexVolumes documentation and parameters:
+  # https://cloud.google.com/anthos-config-management/docs/latest/reference/constraint-template-library#k8spspflexvolumes
+  parameters: {}
+  #     allowedFlexVolumes: #[]
```
Comment (g-awmalik) [inline bundles/psp-v2022/psp-flexvolume-drivers.yaml:45]: Comment blocks? Are they needed?

File: bundles/psp-v2022/psp-selinux-v2.yaml [hunk 3]
```diff
@@ -41,9 +41,11 @@ spec:
...
-  parameters:
-    allowedSELinuxOptions:
-      - level: s0:c123,c456
-        role: object_r
-        type: svirt_sandbox_file_t
-        user: system_u
+  # K8sPSPSELinuxV2 documentation and parameters:
+  # https://cloud.google.com/anthos-config-management/docs/latest/reference/constraint-template-library#k8spspselinuxv2
+  parameters: {}
+  #    allowedSELinuxOptions:
```
Comment (g-awmalik) [inline bundles/psp-v2022/psp-selinux-v2.yaml:47]: If this is left for reference, maybe some description like "to set linux options set the following param as:..." would help.

**Review by g-awmalik — APPROVED** [review]
Looks fine overall except for commented blocks of code in various manifests. Is that there for reference or did we just forget to take it out? If for reference,  additional description/clarity may help the consumer understand the purpose.

File: bundles/psp-v2022/psp-flexvolume-drivers.yaml (hunk 2 above)
Comment (apeabody) [reply bundles/psp-v2022/psp-flexvolume-drivers.yaml:45]: Yeah - They are to help with the Migration workflow: https://cloud.google.com/blog/products/containers-kubernetes/migrate-psp-policies-policy-bundle

File: bundles/psp-v2022/psp-selinux-v2.yaml (hunk 3 above)
Comment (apeabody) [reply bundles/psp-v2022/psp-selinux-v2.yaml:47]: Good call, we'll include in the next revision, currently this is to assist with the Migration workflow: https://cloud.google.com/blog/products/containers-kubernetes/migrate-psp-policies-policy-bundle

**Pull request merged**